File: Data Structures & Algorithms/valid-sudoku/submission-0.java
class Solution {
    public boolean isValidSudoku(char[][] board) {
        HashSet<String> flagger=new HashSet<>();
        for(int r=0;r<board.length;r++){
            for(int c=0;c<board[0].length;c++){
                char num=board[r][c];
                if(num!='.'){
                    if(!flagger.add(num + " is not in the row" +r)
                    ||!flagger.add(num + " is not in the col" +c)
                    ||!flagger.add(num +" is not int the grid" +c/3+"-"+r/3)){
                        return false;
                    }
                }
            }
        }
        return true;
    }
}

File: Data Structures & Algorithms/max-area-of-island/submission-0.py
class Solution:
    def maxAreaOfIsland(self, grid: List[List[int]]) -> int:
        row=len(grid)
        col=len(grid[0])
        maxSize=0
        direc=[[1,0],[-1,0],[0,1],[0,-1]]
        def dfs(r,c):
            if r<0 or c<0 or r>=row or c>=col or grid[r][c]==0:
                return 0
            grid[r][c]=0
            area=1
            for dr,dc in direc:
                area+=dfs(r+dr,c+dc)
            return area

        for r in range(row):
            for c in range(col):
                maxSize=max(dfs(r,c),maxSize)
        return maxSize